paper/code/coffea.py

```diff
@@ -25,6 +25,7 @@ def lepton_filter(lep_charge, lep_type):
     return ak.all([sum_lep_charge == 0, good_lep_type], axis=0)
 
 
+# VK: I think we can skip this. Too much detail and you don't need a processor anymore in coffea.
```

Comment:
Yeah, good point. I was grabbing old code, so we should fix this up for the next draft.